string action
string participant_name
int32 task_number
time stamp

# Actions
string LOAD=load task
string EDIT=edit landmark
string TEST=test landmark
string SAVE=save landmark
